cli/README.md

```diff
@@ -61,12 +61,14 @@ cd /usr/share/FreshRSS
 ./cli/list-users.php
 # Return a list of users, with the default/admin user first
 
-./cli/user-info.php -h --user username
+./cli/user-info.php -h --header --user username1 --user username2 ...
 # -h is to use a human-readable format
-# --user can be a username, or '*' to loop on all users
+# --header outputs some columns headers
+# --user indicates a username, and can be repeated
 # Returns: 1) a * iff the user is admin, 2) the name of the user,
```

Comment:
there is an extra ''f'' in ''iff''. You may consider to change that as well in your PR

Reply:
It was originally on purpose https://en.wikipedia.org/wiki/Iff
but you are not the first one thinking it was an error, so it my be best to remove indeed

Reply:
I probably only know about if and only if because my high school math teacher mentioned it at some point. I wouldn't be inclined to think it was on purpose, generally speaking.

Reply:
I didn't know about that! Thank you for pointing that out.

Reply:
Done :-)